### ISSUE TITLE
How does 'market1501' work as a data set?

### ISSUE THREAD

**Issue by doorteeth** [ZQPei/deep_sort_pytorch#62]
Hi, ZQPei, I`m a newbie. And I don`t know reconstructed data set.
Can you help me? Thank you very much. I agonized over it for two days.
`train_dir = os.path.join(root,"train")`
`trainloader = torch.utils.data.DataLoader(
    torchvision.datasets.ImageFolder(train_dir, transform=transform_train),
    batch_size=64,shuffle=True
)`

**Comment by doorteeth** [comment]
Thanks a lot. I konw it.

**Comment by crazylazydboy** [comment]
Hi, @doorteeth Sorry for bothering you on already closed question.. Were you able run this repo on a video ??

I just want to run a simple video on this but..
after I updated this `self.vdo = cv2.VideoCapture('MyVideo.avi')` and it's still asking me for VIDEO_PATH
